### ISSUE TITLE
merge HD and SD channels into one channel

### ISSUE THREAD

**Issue by revenz** [revenz/NextPvrWebConsole#53]
as a user I would like to merge HD and SD channels into one channel.

For example TV2 SD and TV2 HD should just be TV2.

I would need the ability to set the following options
- Client, which feed is supported (could be variable passed by pincode, eg "user:hd" 
- Recording, if HD feed or SD feed should be used, would need to have option "use feed available to client" so if record scheduled via a SD client, record sd feed, else chose best quality.  But also might want to record HD regardless of client.

**Comment by revenz** [comment]
this must be done before the beta release as it will likely require users to delete their web console database and lose all their user data.

must warn alpha users that during alpha they will likely need to delete their database at some point

**Comment by revenz** [comment]
would also need the ability to unmerge channels, which would mean no breaking change to the database (perhaps an upgrade, but nothing that would break).

**Comment by revenz** [comment]
since not a breaking change, not so urgent, changed to "nice to have"
